api-list.go

```diff
@@ -401,6 +409,14 @@ func (c Client) listObjectsQuery(bucketName, objectPrefix, objectMarker, delimit
 	if err != nil {
 		return listBucketResult, err
 	}
+	for i, content := range listBucketResult.Contents {
+		etag := content.ETag
+		// Trim off the odd double quotes from ETag in the beginning and end.
+		// Note that XML decoder puts two leading and trailing quotes for some reason.
```

Comment:
This comment is not correct @krishnasrinivas - in S3 API it is expected to set `&quot;` .